lib/KImg/__tests__/KImg.spec.js

```diff
@@ -1,92 +1,103 @@
-    const error = jest.fn();
-    makeWrapper({
-      propsData: { src: '/le-logo.svg', altText: 'LE logo', aspectRatio: '16/9' },
-      listeners: { error },
+      it(`sets 'alt' attribute to an empty string`, () => {
+        const wrapper = makeWrapper({
+          propsData: { src: '/le-logo.svg', altText: undefined, isDecorative: true },
+        });
+        expect(wrapper.exists()).toBe(true);
+        expect(wrapper.find('img').attributes('alt')).toBe('');
+      });
     });
-    expect(error).toHaveBeenCalled();
-    expect(error.mock.calls[0][0]).toBeInstanceOf(Error);
-    expect(error.mock.calls[0][0].message).toBe('Invalid aspect ratio provided: 16/9');
-  });
 
-  it(`doesn't throw an error when 'aspectRatio' has a valid format`, () => {
-    const error = jest.fn();
-    makeWrapper({
-      propsData: { src: '/le-logo.svg', altText: 'LE logo', aspectRatio: '16:9' },
-      listeners: { error },
+    it(`throws an error when 'aspectRatio' has an invalid format`, () => {
+      const error = jest.fn();
+      makeWrapper({
+        propsData: { src: '/le-logo.svg', altText: 'LE logo', aspectRatio: '16/9' },
+        listeners: { error },
+      });
+      expect(error).toHaveBeenCalled();
+      expect(error.mock.calls[0][0]).toBeInstanceOf(Error);
+      expect(error.mock.calls[0][0].message).toBe('Invalid aspect ratio provided: 16/9');
     });
-    expect(error).not.toHaveBeenCalled();
-  });
 
-  it(`emits an 'error' event when there is an error in loading the image`, async () => {
-    const error = jest.fn();
-    const wrapper = makeWrapper({
-      propsData: { src: '/le-logo.svg', altText: 'LE logo' },
-      listeners: { error },
+    it(`doesn't throw an error when 'aspectRatio' has a valid format`, () => {
+      const error = jest.fn();
+      makeWrapper({
+        propsData: { src: '/le-logo.svg', altText: 'LE logo', aspectRatio: '16:9' },
+        listeners: { error },
+      });
+      expect(error).not.toHaveBeenCalled();
     });
 
-    // Manually trigger the onError method to simulate the image load failure
-    const e = new Event('error');
-    wrapper.vm.onError(e);
+    it(`emits an 'error' event when there is an error in loading the image`, async () => {
+      const error = jest.fn();
+      const wrapper = makeWrapper({
+        propsData: { src: '/le-logo.svg', altText: 'LE logo' },
+        listeners: { error },
+      });
+
+      // Manually trigger the onError method to simulate the image load failure
+      const e = new Event('error');
+      wrapper.vm.onError(e);
 
-    expect(error).toHaveBeenCalled();
-    expect(error.mock.calls[0][0]).toBeInstanceOf(Event);
-    expect(error.mock.calls[0][0]).toEqual(e);
-  });
+      expect(error).toHaveBeenCalled();
+      expect(error.mock.calls[0][0]).toBeInstanceOf(Event);
+      expect(error.mock.calls[0][0]).toEqual(e);
+    });
+  } else {
+    describe('KImg Visual Tests', () => {
```

Comment:
I think we can clean these changes and keep the KButton instead =)

Reply:
Sure :+1: